media/gpu/v4l2/generic_v4l2_device.cc

```diff
@@ -474,7 +474,7 @@ bool GenericV4L2Device::OpenDevicePath(const std::string& path, Type type) {
     return false;
 
 #if BUILDFLAG(USE_LIBV4L2)
-  if (type == Type::kEncoder &&
```

Comment:
I believe  we can improve is this hunk: 

should not we #if this if condition instead of delete it?

Reply:
Sorry for my shallow understanding.
I have no idea whether it should be delete or not.
Could you tell me why only kEncoder  is allowed?
kDecoder also uses this routing using this patch.
Does it bother if kDecoder is allowed?

Reply:
Kodama-san,

I agree that it makes sense to have this as a define so that the default behavior will match Chromium upstream (the same as we do for the linux_v4l2_only setting).